source/reference/versioning.txt

```diff
@@ -39,21 +39,14 @@ asked to upgrade in order to receive support.
 MongoDB Server Version Compatibility and Support
 ------------------------------------------------
 
-{+c2c-product-name+} supports the following versions of MongoDB Server:
+.. include:: /includes/fact-minimum-server-version-support.rst
 
+``mongosync`` doesn't support MongoDB rapid releases such as 6.1 or 6.2.
 
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
+.. include:: /includes/fact-minimum-versions.rst
 
-   * - ``mongosync`` Version
-     - MongoDB Server Version
+.. include:: /includes/cross-version-sync.rst
 
-   * - 1.6.0
-     - 7.0.x
-
-   * - 1.0.0
-     - 6.0.x
 
 Considerations
 --------------
```

Comment:
Would you mind quickly updating the below to "version-related considerations:" ? Thank you!